docs/sphinx/source/guide-user/installation.rst

```diff
@@ -0,0 +1,96 @@
+************
+Installation
+************
+
+Dependencies
+============
+
+* CMake 3.24+.
+* SYCL 2020: Tested with hipsycl 0.9.4 or Intel DPCPP 2022.1.0) with USM support.
+* MPI 3.0: Tested with MPICH 4.0 or IntelMPI 2021.6. See known issues for Ubuntu 22.03 mpich.
+* HDF5 (parallel): (optional see CMake variable ``NESO_PARTICLES_ENABLE_HDF5``) If particle trajectories are required - will execute without.
+
+Using with CMake 
+================
+
+We provide a CMake interface for projects to access as follows:
```

Comment:
Can we add something about using as a submodule too?
e.g. something like
Alternatively you can add NESO-Particles as a submodule to your project and use
`add_subdirectory(/path/to/NESO-Particles)`
instead of `find_package`

Reply:
Sorry I missed that. Amended that section to cover both ways (I'm not that familiar with the syntax - I copied it from NESO...).

Reply:
Don't worry about it though - unless you object I'll just do it after it's merged.

Reply:
Nope - added that section. Also renamed `NESO_PARTICLES_SYCL_FIND` to `NESO_PARTICLES_FIND_SYCL` such that the variable matches the function name. Don't think I see anything else to change before merging.